addons/l10n_id_efaktur/models/account_move.py

```diff
@@ -64,11 +64,11 @@ def _compute_need_kode_transaksi(self):
     @api.constrains('l10n_id_kode_transaksi', 'line_ids')
     def _constraint_kode_ppn(self):
         ppn_tag = self.env.ref('l10n_id.ppn_tag')
-        for move in self.filtered(lambda m: m.l10n_id_kode_transaksi != '08'):
+        for move in self.filtered(lambda m: m.l10n_id_need_kode_transaksi and m.l10n_id_kode_transaksi != '08'):
```

Comment:
@agr-odoo Should not the constraint then depend on this need_kode_transaksi or the partner_id as well?

Reply:
@jco-odoo Indeed, I added the same depends we have for `_compute_need_kode_transaksi`